masterlist.yaml

```diff
@@ -7889,3 +7889,14 @@ plugins:
         crc: 0x82DAC22D
         util: '[FNVEdit v4.0.4](https://www.nexusmods.com/newvegas/mods/34703)'
         itm: 1
+
+  - name: 'HiddenValleyOverhauled.esm'
+    url: [ 'https://www.nexusmods.com/newvegas/mods/70859/' ]
+    dirty:
+      - <<: *quickClean
+        crc: 0xA2898CFC
+        util: '[FNVEdit v4.0.4](https://www.nexusmods.com/newvegas/mods/347003)'
+        itm: 13
+    clean:
+      - crc: 0x2EF0FD91
+        util: 'FNVEdit v4.0.4'
```

Comment:
We usually don't add `clean` data, if the plugin is `dirty`, so this can be removed.

Reply:
Clean data removed.